2021-03-15 21:48:07,256 INFO    MainThread:11807 [internal.py:wandb_internal():88] W&B internal server running at pid: 11807, started at: 2021-03-15 21:48:07.255934
2021-03-15 21:48:07,257 DEBUG   HandlerThread:11807 [handler.py:handle_request():101] handle_request: run_start
2021-03-15 21:48:07,259 INFO    WriterThread:11807 [datastore.py:open_for_write():77] open: /Users/truongtn/Desktop/Desktop/HocTap/Semester8/Python/YoloV5/yolov5/wandb/offline-run-20210315_214806-5t7op3q6/run-5t7op3q6.wandb
2021-03-15 21:48:09,767 DEBUG   HandlerThread:11807 [meta.py:__init__():34] meta init
2021-03-15 21:48:09,767 DEBUG   HandlerThread:11807 [meta.py:__init__():48] meta init done
2021-03-15 21:48:09,767 DEBUG   HandlerThread:11807 [meta.py:probe():190] probe
2021-03-15 21:48:09,769 DEBUG   HandlerThread:11807 [meta.py:_setup_git():180] setup git
2021-03-15 21:48:09,796 DEBUG   HandlerThread:11807 [meta.py:_setup_git():187] setup git done
2021-03-15 21:48:09,796 DEBUG   HandlerThread:11807 [meta.py:_save_pip():52] save pip
2021-03-15 21:48:09,797 DEBUG   HandlerThread:11807 [meta.py:_save_pip():66] save pip done
2021-03-15 21:48:09,797 DEBUG   HandlerThread:11807 [meta.py:probe():231] probe done
2021-03-15 21:49:04,329 WARNING MainThread:11807 [internal.py:is_dead():343] Internal process exiting, parent pid 11771 disappeared
2021-03-15 21:49:04,330 ERROR   MainThread:11807 [internal.py:wandb_internal():142] Internal process shutdown.
2021-03-15 21:49:04,370 INFO    SenderThread:11807 [sender.py:finish():872] shutting down sender
2021-03-15 21:49:04,522 INFO    WriterThread:11807 [datastore.py:close():258] close: /Users/truongtn/Desktop/Desktop/HocTap/Semester8/Python/YoloV5/yolov5/wandb/offline-run-20210315_214806-5t7op3q6/run-5t7op3q6.wandb
2021-03-15 21:49:04,522 INFO    HandlerThread:11807 [handler.py:finish():535] shutting down handler
2021-03-15 21:49:04,525 INFO    MainThread:11807 [internal.py:handle_exit():78] Internal process exited
